core/core_modules/update.js

```diff
@@ -37,24 +37,33 @@ var git = require.once('../git.js'),
 
 				if (err) {
 					failedUpdateAttempts++;
+					console.critical(err);
 					console.debug('Periodic auto update failed. Manual intervention is probably required.\n Error: ' + consoleOutput);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2554993048/issues/source?bid=3282987&fileBranchId=3284644#l41)